docs/source/workbench/messageswindow.rst

```diff
@@ -4,15 +4,32 @@
 Messages Window
 ===============
 
-All of the output from your scripts and algorithm's ran from interfaces or other places will be output here. This is
-alongside the ability change the logging level. The logging level has 5 different options to display the right level of
-detail to choose the correct amount of detail for your circumstances.
+All of the output from your scripts and algorithms run from interfaces or
+other places will be output here. This is alongside the ability change the
+logging level. The logging level has 5 different options to display the right
+level of detail to choose the correct amount of detail for your circumstances.
 
 .. image:: ../images/Workbench/MessageWindow/MessagesWidget.png
     :align: right
     :scale: 50%
 
-Logging Level
--------------
+All output from running your scripts or Mantid algorithms is logged in
+this window. To minimise the amount of information displayed in this window you
+are provided with 5 logging levels (in ascending order of priority):
```

Comment:
Would it be possible to describe the type of people/content that may be interested/is present at different logging levels?

Reply:
I've added this in commit 5dad4a4. Also `To minimise the amount of information displayed in this window..` isn't the right phrase here, I'll change that in my next commit.